docs/scenarios/v2x.rst

```diff
@@ -18,24 +18,31 @@ Build V2X module
 
 .. code-block:: bash
 
-   cd zenoh_autoware_v2x && poetry install --no-root
+   cd autoware_carla_launch
+   source env.sh
+   cd external/zenoh_autoware_v2x
+   poetry env use $(pyenv which python) && poetry install --no-root
 
 * Enter Autoware container
 
 .. code-block:: bash
 
-   cd zenoh_autoware_v2x && colcon build --symlink-install
-   pip install eclipse-zenoh==0.10.0rc0 
+   cd autoware_carla_launch/external/zenoh_autoware_v2x
+   pip install -r requirements.txt
```

Comment:
I believe you don't need this anymore since you've already installed them in Dockerfile